Update example config with UDP precision option

gunnaraasen opened the pull request:
###### Required for all non-trivial PRs
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] Config changes: update sample config (`etc/config.sample.toml`), server `NewDemoConfig` method, and `Diagnostics` methods reporting config settings, if necessary

This updates the config and diagnostic info for the UDP service to include the `precision` option, which was added in PR https://github.com/influxdata/influxdb/pull/5565.

rbetts commented:
Before we merge this documentation change, it looks like the original PR didn't include a testcase. Also, we need to review this for Enterprise in case this is a configuration stanza that needs to be updated in both places.

mark-rushakoff commented:
Yep, looks like services/udp/service_test.go still hasn't been updated with a precision test case since that original PR.  This PR so far looks good, but +1 to adding a test case.

gunnaraasen commented:
Enterprise already has precision documented in the sample config file. I think not including it here was an oversight.

I've updated the text to reflect the Enterprise config snippet and also added precision to the UDP config test.
